src/general_physics.py

```diff
@@ -0,0 +1,28 @@
+from dataclasses import dataclass
+from src.common_constants import CommonConstants
+from src.vec2d import Vec2d
+
+
+@dataclass(frozen=True)
+class GeneralPhysics:
```

Comment:
Is calculating pixels a physics problem? Why not call it "size convertion" or similar?

Reply:
It Comes into Play when doing physical calculations, to make these better to grasp intuitively. Apart from that, we could also set everything in pixels

Reply:
Im still curious how translating between real and digital world will work, but i'm sure you have a good solution in your mind.